okapi/services/caches/formatters/gpxfile.tpl.php

```diff
@@ -11,7 +11,7 @@
 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 xsi:schemaLocation="
 http://www.topografix.com/GPX/1/0 http://www.topografix.com/GPX/1/0/gpx.xsd
-http://www.opencaching.com/xmlschemas/opencaching/1/0 http://www.opencaching.com/xmlschemas/opencaching/1/0/opencaching.xsd
+http://www.opencaching.com/xmlschemas/opencaching/1/0 https://github.com/opencaching/okapi/blob/master/etc/nsox.xsd
```

Comment:
I think that you need a raw URL here. The one you have here will output HTML (not XML Schema), so the validators won't recognize it.